src/App.jsx

```diff
@@ -1,9 +1,31 @@
 import './App.css'
+import Button from "./components/Button.jsx";
+
+function clickHandler(e) {
```

Comment:
deze function kan je voor de netheid in je button zetten. Aangezien het past bij de functie van je knopje.

Reply:
Dan hoef je niet een extra functie aan te roepen in je button